src/main/groovy/nextflow/validation/config/ValidationConfig.groovy

```diff
@@ -4,30 +4,63 @@ import groovy.util.logging.Slf4j
 
 import nextflow.validation.exceptions.SchemaValidationException
 
+import nextflow.config.schema.ConfigOption
+import nextflow.config.schema.ConfigScope
+import nextflow.config.schema.ScopeName
+import nextflow.script.dsl.Description
+
 /**
- * This class allows model an specific configuration, extracting values from a map and converting
+ * This class is used to read, parse and validate the `validation` config block.
  *
  * @author : nvnieuwk <nicolas.vannieuwkerke@ugent.be>
  *
  */
 
 @Slf4j
+@ScopeName('validation')
+@Description('''
+    The `validation` scope allows you to configure the `nf-schema` plugin.
+''')
 class ValidationConfig {
```

Comment:
Every config scope class needs to implement `nextflow.config.schema.ConfigScope`, this one and the nested scope classes. This is why it's not being picked up by Nextflow